src/js/themes/hpe.js

```diff
@@ -2215,6 +2350,7 @@ const buildTheme = (tokens, flags) => {
     select: {
       clear: {
         container: {
+          background: 'transparent',
```

Comment:
Is this change intentional?

Reply:
Yeah intentional -- when I was testing locally, the base.js grey background was still coming through on the container, but we don't want that. (related to the select clear container PR that went into NEXT)

Reply:
Cool, thanks for confirming